src/instruments/src/Clock/Components/DayNight.tsx

```diff
@@ -0,0 +1,15 @@
+import { useSimVar } from '@instruments/common/simVars';
+
+const secondsToDisplay = (seconds: number): number[] => {
+    const displayTime = [0, 0, 0];
+    displayTime[0] = Math.floor(seconds / 3600);
+    return displayTime;
+};
+
+const DayNight = () => {
+    const [localTime] = useSimVar('E:LOCAL TIME', 'seconds', 200);
+    const displayTime = secondsToDisplay(localTime);
+    return ((displayTime[0] >= 6 && displayTime[0] <= 18) ? 'day' : 'night');
+};
```

Comment:
You can't do this. Either DayNight is a React component, or it's not. If it is, you can use hooks, but you need to actually render and return JSX, not a string. If it is not (and it's just a normal function), you cannot use hooks. [These are the rules of hooks](https://reactjs.org/docs/hooks-rules.html).

To fix this you could make DayNight an actual function like `getDayNightClassName` and pass it localTime as a parameter.

Reply:
@beheh Thanks for the feedback! As you can see with this goof up above, I'm fairly new to React. 😄 I'll look into it and make the suggested changes

Reply:
>  [These are the rules of hooks](https://reactjs.org/docs/hooks-rules.html).

I'm in the same boat as sidnov. This page helped explain some things I saw, so thank you. 👍